src/server/admin/tags/helpers/service-tags.js

```diff
@@ -0,0 +1,47 @@
+import { renderComponent } from '~/src/server/common/helpers/nunjucks/render-component.js'
+import Joi from 'joi'
+
+/**
+ * List of tags that can be applied to a service.
+ * @type {Map<string, {colour: string, displayName: string, name: string, description: string}>}
+ */
+export const serviceTags = {
+  'tier-1': {
+    name: 'tier-1',
+    displayName: 'Tier 1',
+    description: 'Service has Tier 1 support coverage.',
+    colour: 'purple'
+  },
+  live: {
+    name: 'live',
+    displayName: 'Live',
+    description: 'Service has passed its Platform Readiness Review',
+    colour: 'green'
+  }
+}
+
+export const validServiceTags = Joi.string().valid(...Object.keys(serviceTags))
+
+/**
+ * Returns an HTML representation of a tag.
+ * Takes either the name of a valid tag or a tag object.
+ * If the tag name is invalid it returns null.
+ * @param {string|{name:string, displayName:string, description:string, colour: string}} tagOrName
+ * @returns {string|null}
+ */
+export function renderServiceTag(tagOrName) {
+  const tag =
+    typeof tagOrName === 'string'
+      ? serviceTags[tagOrName.toLowerCase()]
+      : tagOrName
+
+  if (!tag) {
+    return null
+  }
+
+  return renderComponent('tag', {
```

Comment:
we already have https://github.com/DEFRA/cdp-portal-frontend/blob/a4edb6f05cc8b7ed0395cc7d14f1f5bc0088aa84/src/server/admin/permissions/helpers/render-tag.js